docs/ADMINISTRATION_AND_MAINTENANCE_PLAN.md

```diff
@@ -0,0 +1,21 @@
+# Our maintenance plan
+
+This roadmap outlines responsibilities for administration of funds in the DCAF-administered Heroku pipeline (e.g. where DCAF administers DARIA for them), and roadmapping new development.
+
+## Administration
+
+* The engineering team has set up a series of monitors for applications, and is responsible for keeping an eye on them and troubleshooting when something goes wrong.
+* The engineering team is responsible for applying security fixes and other regular maintenance.
+* The DCAF case management team is responsible for communicating support requests to the engineers in the slack channel.
+* We can allow fund stakeholders access to management of the application so they can restart it if they like; funds should reach out to the engineering team or DARIA project lead if they're interested.
+
+## Developing new features
+
+* Every once in awhile (probably six to nine months), the DARIA project lead and DCAF data board member will poll DCAF and other member funds and see what development priorities or pain points have popped up, and select one or two priorities to work on.
+* When this happens, the DARIA project lead is responsible for assembling engineers and pushing it through.
+* A core team of engineers will continue to review pull requests.
+
+## Getting new funds on DARIA
```

Comment:
Do we have another writeup on this we can link to? If not, can we explain that it's relatively painless?

Reply:
Found it: https://github.com/DCAFEngineering/dcaf_case_management/blob/master/docs/SETTING_UP_A_NEW_INSTANCE.md#by-the-way

Reply:
Added a link and an additional bullet in 70d5f2f4 . This is a great call.